src/request.ts

```diff
@@ -152,6 +169,108 @@ export async function getRequestLinkDetails({
 	return x
 }
 
```

Comment:
add comment. imagine someone new integrating peanut-sdk wants to use this function - what would you tell them?